wasi/wasi.go

```diff
@@ -277,16 +323,28 @@ func (w *WASIEnvironment) fd_close(ctx *wasm.HostFunctionCallContext, fd uint32)
 	return ESUCCESS
 }
 
-func args_sizes_get(ctx *wasm.HostFunctionCallContext, argcPtr uint32, argvPtr uint32) (err Errno) {
-	// not implemented yet
-	binary.LittleEndian.PutUint32(ctx.Memory.Buffer[argcPtr:], 0)
-	binary.LittleEndian.PutUint32(ctx.Memory.Buffer[argvPtr:], 0)
-	return 0
+func (w *WASIEnvironment) args_sizes_get(ctx *wasm.HostFunctionCallContext, argsCountPtr uint32, argsBufSizePtr uint32) Errno {
```

Comment:
This file isn't maintained, yet, but needs to have godoc on each new method (until someone backfills the others) that summarizes what's going on and cites a link to the spec we are following (permalink to a specific pre-release of wasi vs a master link if available)

Reply:
Added docs. PTAL